src/game/boe.actions.cpp

```diff
@@ -780,7 +780,12 @@ static void handle_use_space(location destination, bool& did_something, bool& ne
 	put_item_screen(stat_window);
 }
 
-static void handle_bash_pick_select(bool& need_reprint, bool isBash) {
+void handle_bash_pick_select(bool& need_reprint, bool isBash) {
+	if(recording){
+		std::ostringstream sstr;
+		sstr << std::boolalpha << isBash;
+		record_action("handle_bash_pick_select", sstr.str());
```

Comment:
Instead of a boolean, would it maybe be better for this to be two different actions?

Reply:
I was thinking the same thing while I wrote this code. I'll do that.